### ISSUE TITLE
Use a custom field to send the mail to in Elementor Forms

### ISSUE THREAD

**Issue by adorbit** [elementor/elementor#13079]
Hi there!

I have a website that has an emailaddres added as a custom field to a post (or page). The emailaddress can be different on every page. 

On that page is an Elementor form. I can fill the formfields default value using custom fields but currently it's not possible to use custom fields in the **email action** (where to send the mail to). Does anyone know if this is possible?

I added a screenshot to make the idea clear:
<img width="410" alt="elementor-screen0" src="https://user-images.githubusercontent.com/63058871/99281770-33a7a500-2833-11eb-8828-42cdd2b4f79f.png">

Thanks in advance!

Cheers,
Rob / The Netherlands

**Comment by haniefss** [comment]
you could load that custom field to a hidden field (let's say the hidden field ID is **hidden_field_email**)
set [field id="hidden_field_email"] in email action as destination address (to).

![image](https://user-images.githubusercontent.com/39113199/99463052-7ff40180-2967-11eb-95c2-70cd27a27359.png)

**Comment by adorbit** [comment]
Thanks for the suggestion!

I already tried that but if it's a hidden field, the e-mail address is still visible in the HTML source. It would be much cleaner to get the address straight from the custom field.

**Comment by Luxamman** [comment]
Yes please, waiting for that feature for about two years now... Thanks!
